lib/crawly/utils.ex

```diff
@@ -224,4 +224,27 @@ defmodule Crawly.Utils do
   def ensure_loaded?(module) do
     Code.ensure_loaded?(module)
   end
+
+  def unwrap_module_and_options(
+        setting,
+        spider_name \\ nil,
+        default_value \\ nil
+      ) do
+    case Crawly.Utils.get_settings(
+           setting,
+           spider_name,
+           default_value
+         ) do
+      {module, args} when is_list(args) and is_atom(module) ->
+        {module, args}
+
+      module when is_atom(module) ->
+        {module, nil}
+
+      {module} ->
```

Comment:
switch this to a `_` so that it'll match on anything.

Reply:
used `x` so I can use the value in `inspect/1`